apps/swirl-docs/src/components/Grid/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { FunctionComponent } from "react";
+
+export type GridProps = {
+  data: any;
+  className?: string;
+  children: JSX.Element | JSX.Element[];
+};
+
+export const Grid: FunctionComponent<GridProps> = ({ className, children }) => {
+  return (
+    <div role="grid" aria-colcount={1} aria-rowcount={1}>
```

Comment:
colcount and rowcount seem unnecessary, as all columns and rows are visible all the time.